code-challenges.test.js

```diff
@@ -37,9 +69,32 @@ const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
+describe("theLetter", () => {
+    it("takes in an array of words and a single letter and returns all the words that contain that particular letter.", () => {
+      const arrayOfWords1 = ["Apple", "Banana", "Plum", "Orange", "Kiwi"]
+      const letterA = "a"
+      const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
+      const letterE = "e"
+      expect(theLetter(arrayOfWords1)).toEqual(expect.arrayContaining(["Apple", "Banana", "Orange"]))
+      expect(theLetter(arrayOfWords2)).toEqual(expect.arrayContaining(["Cherry", "Blueberry", "Peach"]))
+    })
+  })
+
+//   ReferenceError: theLetter is not defined
 
 // b) Create the function that makes the test pass.
 
+// If the string contains either "a" or "e" return the string.
+
+const theLetter = (array) => {
+    return array.filter(value => {
+      if (value "a"){
+        return string
+      }
+      if (value "e"){
+        return string
+    })
+  }
```

Comment:
We are missing some pieces to the puzzle. You should look into multiple arguments for functions.